omnigibson/prims/joint_prim.py

```diff
@@ -866,6 +867,9 @@ def keep_still(self):
         Zero out all velocities for this prim
         """
         self.set_vel(np.zeros(self.n_dof))
+        # If not driven, set torque equal to zero as well
+        if not self.driven:
+            self.set_effort(np.zeros(self.n_dof))
```

Comment:
why not always set torque to 0?

Reply:
looping in @cgokmen . You mentioned we actually shouldn't set efforts directly now?